src/ui/chat-window.js

```diff
@@ -298,6 +327,17 @@ export class ChatScene extends Phaser.Scene {
                         this.enemyStatText[varName].text = object[varName];
                         this.enemyStatText[varName].visible = true;
                     }
+
```

Comment:
Enemy stat defenses aren't formatted nicely. Examine a cow and you'll see what I mean

Reply:
Yeah it's double digit and negative so it goes over. There's not a lot more space to work with there so suggestions welcome

Reply:
Use a second line!

Reply:
You mean have the numbers under the text?
like:
```
Attack  Strength  Defense  Magic  Ranged
1       1         1        1         1
Melee                      Magic  Ranged
-15                          0         0
...
```
Would basically double the line count, I don't think that'd fit:
![image](https://user-images.githubusercontent.com/9402065/88463466-933f0680-ce78-11ea-9ba7-de58caceefc9.png)

Reply:
https://github.com/CurtisGreen/rs-clicker/pull/6